### PR TITLE
allow 3.6.x sdks

### PR DESCRIPTION
**Pull request by jakemac53**
unblock the version bump in the SDK

### CODE REVIEW

File: build_modules/pubspec.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+version: 5.0.9
 description: >-
   Builders to analyze and split Dart code into individually compilable modules
   based on imports.
 repository: https://github.com/dart-lang/build/tree/master/build_modules
 resolution: workspace
 
 environment:
-  sdk: '>=3.5.0-259.0.dev <3.6.0'
+  sdk: '>=3.5.0-259.0.dev <3.7.0'
```
Comment (natebosch) [inline build_modules/pubspec.yaml:10]: Is there any reason not to change the lower bound to `>=3.5.0`?

**Review by natebosch — APPROVED** [review]

File: build_modules/pubspec.yaml (hunk 1 above)
Comment (jakemac53) [reply build_modules/pubspec.yaml:10]: Well, 3.5 isn't actually out yet.

Comment (jakemac53) [reply build_modules/pubspec.yaml:10]: At a minimum our "pubspec" SDK jobs would fail.

Comment (natebosch) [reply build_modules/pubspec.yaml:10]: Ah, surprising that we're prepping for 3.6 without launching 3.5...

LGTM.

Comment (jakemac53) [reply build_modules/pubspec.yaml:10]: Yeah it happens when we cut the branch that will be 3.5 stable. That way we don't have misleading -dev tags that make people put in bad constraints.

**Pull request merged**